src/lammpsio/snapshot.py

```diff
@@ -437,6 +442,114 @@ def has_mass(self):
         """
         return self._mass is not None
 
+    @property
+    def bonds(self):
+        """:class:Bonds: bonds."""
+        if not self.has_bonds():
+            self._bonds = None
+        return self._bonds
+
+    @bonds.setter
+    def bonds(self, value):
+        if value is not None:
+            if not isinstance(value, Bonds):
+                raise ValueError("bonds must be Bonds")
+            self._bonds = value
+        else:
+            self._bonds = None
+
+    def has_bonds(self):
+        """Check if configuration has bonds.
+
+        Returns
+        -------
+        bool
+            True if bonds have been initialized.
+
+        """
+        return self._bonds is not None
```

Comment:
Should we also check that the number of bonds is not zero?
```suggestion
        return self._bonds is not None and self._bonds.N > 0
```